backbone/src/main/java/org/researchstack/backbone/ui/step/body/TextQuestionBody.java

```diff
@@ -27,13 +27,13 @@
     //-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
     // Constructor Fields
     //-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
-    private QuestionStep step;
-    private StepResult<String> result;
+    protected QuestionStep step;
```

Comment:
Do we need to increase visibility? Should we make the step and result fields final?

Reply:
Yes, sub-classes should have access to these.